src/index.html

```diff
@@ -7,8 +7,30 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#home" class="logo"><img src="images/logo.png" alt="logo"></a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#apple" class="nav__link is-active">Apple</a></li>
```

Comment:
Remember about code style 
![image](https://github.com/mate-academy/layout_moyo-header/assets/64750958/8864fba6-c990-4018-922d-3837ac6d25ef)
```suggestion
          <li class="nav__item"><a href="#apple" class="nav__link is-active">Apple</a></li>
```